docs/diagrams/evidence-request-creation-sequence-flow.iuml

```diff
@@ -0,0 +1,18 @@
+@startuml
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/v2.2.0/C4_Deployment.puml
+!include https://raw.githubusercontent.com/LBHackney-IT/cv-19-res-support-v3/development/docs/diagrams/c4_shared.puml
+
+title Evidence request creation sequence flow diagram
+
+Officer -> Next.JS: Search for resident
+Next.JS -> EvidenceAPI: GET /residents/:residentId
+Next.JS <- EvidenceAPI: 200 Resident found
+Officer <- Next.JS: Resident found
+Officer -> Next.JS: Request evidence
+Next.JS -> EvidenceAPI: POST /evidence_requests
+EvidenceAPI -> EvidenceAPI: Create evidence request in database
+EvidenceAPI -> GovNotify: Send email to resident with link
```

Comment:
As we can also sends SMS maybe we could rephrase to `Send notification`?

Reply:
👍